src/plots/stackedBarPlot.ts

```diff
@@ -148,7 +148,7 @@ export class StackedBar<X, Y> extends Bar<X, Y> {
       if (maximum.extent !== baselineValue) {
         // only draw sums for values not at the baseline
 
-        const text = this.labelFormatter()(maximum.extent);
+        const text = this.labelFormatter()(maximum.extent, maximum.axisValue, maximum.datasets);
```

Comment:
As pointed out above, the extent is computed using multiple datasets. I've attached them to the `StackExtent` but they could be moved to `GenericStackedDatum<D>` instead. In that case, above this line I'd have to add something like:
```typescript
let extentDatasets: Dataset[] = [];
this._stackingResult.forEach((stackedDatum, dataset) => {
    extentDatasets.push(stackedDatum.get(maximum.axisValue).dataset);
});
```